spec/requests/session_controller_spec.rb

```diff
@@ -1188,7 +1188,9 @@ def login_with_sso_and_invite(invite_key = invite.invite_key)
         "*|secret,forAll",
         "*.rainbow|wrongSecretForOverRainbow",
         "www.random.site|secretForRandomSite",
+        "somewhere.over.rainbow|oldSecretForOverRainbow",
```

Comment:
Was wondering if we need this since we already have `secretForOverRainbow` below

Reply:
I wanted the overlap secret being tested to be the "middle" secret - not last or first in case there was a bug in the secret matching (eg first always works or last always works)